docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -462,6 +462,10 @@ The following request returns a JSON document containing pipeline stats,
 including:
 
 * the number of events that were input, filtered, or output by each pipeline
+* the current and lifetime _flow_ rates for useful pipeline metrics:
+** `input_throughput`, `filter_throughput`, and `output_throughput`, expressed in _events-per-second_
+** `worker_concurrency`, expressed as a ratio of worker execution-time to wall-clock time
+** `queue_backpressure`, expressed as a ratio of time inputs are blocked pushing to the queue to wall-clock time (_NOTE: magnitude of this flow rate is heavily dependent on pipeline structure and is not comparable across pipelines_)
```

Comment:
@karenzone I am not sure here is the best place to document the _meaning_ of our flow-metrics long-term; it is certainly more detail than any of the other bullet-points in the list.

Can you help me find a better home for this granularity?

For context, this is in the scope of #14463, in which we introduce "flow" metrics to represent rates-of-change in addition to the current cumulative-value, to make it easier to see how a pipeline is performing _now_ relative to how it has been performing _before_.
 - In the initial release, we are first adding 5 "flow" metrics to _pipelines_, each of which are the rate of change of a particular pipeline metric with respect to time passing, and each of the flow metrics will present a `lifetime` rate, and a `current` rate.
 - Subsequent iterations will add more rolling averages to each flow metric (so we'll continue to have `current` and `lifetime`, but will also have something like `last_1h` once the pipeline has been alive for more than an hour; the precise periods that are covered by these rolling averages is to-be-determined and will likely come with a Technology Preview while we settle on the "right" granularity)
 - Subsequent iterations will also add flow metrics to other things, like PQ metrics, the process as a whole, and even individual plugins.
 - Future flow metrics that we add _may_ use something other than (wall-clock) time as their denominator, such as an individual plugin's throughput using the amount of time that particular plugin has spent in execution.